### PR TITLE
sound: A slightly more comprehensive update in the sandbox folder

### PR DESCRIPTION
**Pull request by jiaoboxuan**
I did a slight update in the sandbox folder in src/sound. I included a few extra functions and also pushed the music files into this branch for the sake of ease of the teams wanting to test it out next year (last year's team had the audio directory in their local files, and they didn't push it up here, so it was kind of a hassle).

### CODE REVIEW

File: src/sound/sandbox/main.c [hunk 1]
```diff
@@ -74,8 +81,8 @@ int main(int argc, char** argv)
 
     SDL_CreateWindow("Sounds", SDL_WINDOWPOS_UNDEFINED,
                      SDL_WINDOWPOS_UNDEFINED,
-                     256,
-                     256,
+                     512,
```
Comment (jwschroeder21) [inline src/sound/sandbox/main.c:84]: Why did you change this to 512?

File: src/sound/sandbox/main.c [hunk 2]
```diff
@@ -106,6 +114,12 @@ int main(int argc, char** argv)
                 case 'e':
                     Mix_PlayMusic(_music[0], 1);
                     break;
+                case ' ':
```
Comment (jwschroeder21) [inline src/sound/sandbox/main.c:117]: I think this is so much cleaner and very well done. Nice job!

File: src/sound/sandbox/main.c (hunk 1 above)
Comment (jiaoboxuan) [reply src/sound/sandbox/main.c:84]: scrambling around the window a little bit. Before, it pops a 256*256 black window; now it pops a 512*512. I just want to make sure I understand what's depended on this function.

Comment (jwschroeder21) [reply src/sound/sandbox/main.c:84]: Okay, I see. Thank you for the clarification!

**Review by ecoble — APPROVED** [review]
Nice work cleaning this up and adding the sound files!

**Review by jwschroeder21 — APPROVED** [review]

**Comment by sangerr** [comment]
Summary: In this pull request we added some functions to the sandbox folder to make it easier to play around and develop new ideas. These functions included Mix_PauseMusic and Mix_ResumeMusic. Overall, this resulted in a cleaner set of functions to work with for us and future teams

**Comment by ecoble** [comment]
Issue Score: Excellent

Comments:
Great job cleaning up this sandbox example and getting it working! And nice job on a thorough review as well!
